Labels.md

```diff
@@ -0,0 +1,79 @@
+# Labels
+
+Various labels are used to decorate objects provided by this project to help end users find the right resources for them.
+
+For example, to list all cluster instance types deployed in the current env that provide `4` vCPUs a user can use the following command:
+
+```shell
+$ kubectl get virtualmachineclusterinstancetype -linstancetype.kubevirt.io/cpu=4
+NAME         AGE
+cx1.xlarge   39s
+gn1.xlarge   39s
+m1.xlarge    39s
+n1.large     39s
+n1.medium    39s
+o1.xlarge    39s
+u1.xlarge    39s
+```
+
+## Generic
+
+### `instancetype.kubevirt.io/version`
+
+The version of common-instancetypes the resources has been generated from.
```

Comment:
This isn't correct, that's the version of the instance type class.

I'm also missing common-instancetypes-version.